README.md

```diff
@@ -1,69 +1,50 @@
 
-This is a demo for the [iv4XR testing framework](https://github.com/iv4xr-project/aplib), demonstrating that iv4XR test agents can control a game called _Space Engineers_ to perform some testing tasks. This repository is a fork of the [*Lab Recruits* demo](https://github.com/iv4xr-project/iv4xrDemo).
+This is a demo for the [iv4XR testing framework](https://github.com/iv4xr-project/aplib), demonstrating that iv4XR test agents can control [_Space Engineers_](https://www.spaceengineersgame.com/) (a game by [Keen Software House](https://www.keenswh.com/)) to perform some testing tasks. This repository started as a fork of the [*Lab Recruits* demo](https://github.com/iv4xr-project/iv4xrDemo), but has been significantly modified since.
 
-It is not intended for general uses yet, other than as a testing project for the development of the [Space Engineers iv4XR plugin](https://github.com/iv4xr-project/iv4xr-se-plugin). For more details please refer to the plugin repository README. 
+It is not intended for general uses, other than as a testing project for the development of the [Space Engineers iv4XR plugin](https://github.com/iv4xr-project/iv4xr-se-plugin). For more details, please refer to the plugin repository README. 
 
 <img src="./docs/SE-sotf1.png" width="100%">
 
-# Original README content below
+# Setup
 
-This is a demo version 2.x for the [iv4XR Agent-based Testing Framework](https://github.com/iv4xr-project/aplib),
-demonstrating that iv4XR test agents can control a game called _Lab Recruits_ to perform testing tasks.
-The game executable is no longer included in the repository. It might be included in certain releases, or else you need to build it yourself
-from its [repository](https://github.com/iv4xr-project/labrecruits).
+## Requirements
 
-You will need correct versions of the iv4XR Framework and the game Lab Recruits, which are compatible with this version of Demo. Check the `pom.xml` to know which versions are needed.
+Project requires [aplib](https://github.com/iv4xr-project/aplib) dependency. First build `aplib` and install it to Maven local.
+Other possibility at the moment is to change dependency do JitPack:
 
-   <img src="./docs/LRSS1.png" width="48%"><img src="./docs/LRSS3.png" width="50%">
+```
+implementation 'com.github.iv4xr-project:aplib:v1.2.0'
+```
 
-**Work in progress notice.** Keep in mind that the work here is still in progress. Things may look ugly while we are working on them, and things may change.
+## Clone repo:
 
-**What is the demo?** A set of JUnit test classes demonstrating how iv4xr test agents are used to implement a number of testing tasks for the game _Lab Recruits_. These classes are located in `src/test/java/agents/demo`. You can simply run them, or modify them yourself. When you run them you don't usually see anything because the tests run pretty fast. You can insert pause-points yourself (e.g. using console-read).
+```
+git clone git@github.com:iv4xr-project/iv4xrDemo-space-engineers.git
+```
```

Comment:
This is kind of obvious. Why is it here? Is it just for easy copy-paste to cmd line? Am I missing something?

Reply:
Exactly, easy copy-paste.

Reply:
Hmm, I find it confusing. You have the link in the corner of every repository on GitHub. Also many people use Gui tools nowadays.

Reply:
That's the thing. You use GUI tools, then you find it useless. I on the other hand use command line a lot and I always appreciate when it is there. I can get the point where it takes space and we want to keep readme file smart, but how exactly is this line confusing for anyone?